DASH-IF-IOP.bs.md

```diff
@@ -10,9 +10,1003 @@ Live discussion in #document-authoring on Slack.
+
+1. The MPD consists of consecutive [=periods=] which map data onto the MPD timeline.
+1. Each period contains of one or more [=representations=], each of which provides media samples inside a sequence of [=media segments=].
+1. Representations within a period are grouped in [=adaptation sets=], which associate related representations and decorate them with metadata.
+
+<figure>
+	<img src="Images/Timing/BasicMpdElements.png" />
+	<figcaption>The primary elements described by an MPD.</figcaption>
+</figure>
+
+The chapters below explore these relationships in detail.
+
+## Periods ## {#timing-period}
+
+An MPD SHALL define an ordered list of one or more <dfn title="period">periods</dfn>. A period is both a time span on the [=MPD timeline=] and a definition of the data to be presented during the period. Period timing is relative to the zero point of the [=MPD timeline=].
+
+<figure>
+	<img src="Images/Timing/PeriodsMakeTheMpd.png" />
+	<figcaption>An MPD is a collection of periods, starting from a zero point and with a total duration (which may be unlimited for live services).</figcaption>
+</figure>
+
+Common reasons for defining multiple periods are:
+
+* Assembling a presentation from multiple self-contained pieces of content.
+* Inserting ads in the middle of existing content and/or replacing spans of existing content with ads.
+* Adding/removing certain representations as the nature of the content changes (e.g. a new title starts with a different set of offered languages).
+* Updating period-scoped metadata (e.g. codec configuration or DRM signaling).
+
+Periods are self-contained - a service SHALL NOT require a client to know the contents of another period in order to correctly present a period. Knowledge of the contents of different periods MAY be used by a client to achieve seamless period transitions, especially when working with [[#timing-connectivity|period-connected representations]].
+
+All periods SHALL be consecutive and non-overlapping. A period MAY have a duration of zero.
+
+Note: A period with a duration of zero might, for example, be the result of ad-insertion logic deciding not to insert any ad.
+
+<div class="example">
+The below MPD example consists of two 20-second periods. The duration of the first period is calculated using the start point of the second period. The total duration is also listed, as it is required for static MPDs.
+
+<xmp highlight="xml">
+<MPD xmlns="urn:mpeg:dash:schema:mpd:2011" type="static" mediaPresentationDuration="PT40S">
+	<Period>
+		...
+	</Period>
+	<Period start="PT20S" duration="PT20S">
+		...
+	</Period>
+</MPD>
+</xmp>
+
+Parts of the MPD structure that are not relevant for this chapter have been omitted - this is not a fully functional MPD file.
+</div>
+
+In a static MPD, the first period SHALL start at the zero point of the [=MPD timeline=]. In a dynamic MPD, the first period SHALL start at or after the zero point of the [=MPD timeline=].
```

Comment:
Should usage of `Period@start` attributes be encouraged or discouraged in different MPD scenarios?
For instance, in a static presentation, the `Period@start` values are redundant given the `Period@duration` and in the case of remote periods `Period@start` will not be correct. In a dynamic presentation, as you want all periods to be fixed within the availability timeline it may make sense to always have `Period@start` present.